orderby_parser.go

```diff
@@ -16,15 +16,11 @@ type OrderByItem struct {
 	Order string            // Ascending or descending order.
 }
 
-func ParseOrderByString(ctx context.Context, orderby string) (*GoDataOrderByQuery, error) {
```

Comment:
This would be an API breaking change. We will need to change the major number if we do that.